torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1094,8 +1098,9 @@ XLATensor XLATensor::index_put(
     tensorflow::gtl::ArraySlice<const XLATensor> indices,
     const XLATensor& values, bool accumulate,
     tensorflow::gtl::ArraySlice<const xla::int64> result_permutation) {
-  return input.CreateFrom(IndexPutByTensors(input, indices, values, accumulate,
-                                            result_permutation));
+  return input.CreateFrom(
```

Comment:
This should be the same as well.
```
x.CreateFrom(y);
```
Expands to:
```
x.CreateFrom(y, x.GetDevice(), x.dtype());
```

Same thing in the other places.

Reply:
Done.